pkg/util/maven/maven_command.go

```diff
@@ -185,7 +185,7 @@ func (c *Context) AddSystemProperty(name string, value string) {
 	c.AddArgumentf("-D%s=%s", name, value)
 }
 
-func generateProjectStructure(context Context, project Project) error {
+func generateProjectStructure(context Context, project *Project) error {
```

Comment:
Out of curiosity, what's the reason of this change? what's the gain? I understand we're going to create a new Project struct in this way, right?

Reply:
The opposite, this way we are passing a pointer so no copy is performed when invoking this method but I reverted it as it is enough to pass by value in this case. It was a leftover of a change to make the methods on the `Project` struct consistently using pointer receiver instead of a mix of pointer and value receivers.

Reply:
Yes, indeed. I think I inverted the red and the green while reading.